src/generated/v3/entityService/entityServiceComponents.ts

```diff
@@ -105,28 +95,84 @@ export type ProjectPitchIndexResponse = {
   }[];
 };
 
-export type ProjectPitchIndexVariables = {
-  queryParams?: ProjectPitchIndexQueryParams;
+export type ProjectPitchesIndexVariables = {
+  queryParams: ProjectPitchesIndexQueryParams;
 };
 
-export const projectPitchIndex = (variables: ProjectPitchIndexVariables, signal?: AbortSignal) =>
+export const projectPitchesIndex = (variables: ProjectPitchesIndexVariables, signal?: AbortSignal) =>
   entityServiceFetch<
-    ProjectPitchIndexResponse,
-    ProjectPitchIndexError,
+    ProjectPitchesIndexResponse,
+    ProjectPitchesIndexError,
     undefined,
     {},
-    ProjectPitchIndexQueryParams,
+    ProjectPitchesIndexQueryParams,
     {}
   >({ url: "/entities/v3/projectPitches", method: "get", ...variables, signal });
 
-export type ProjectPitchGetPathParams = {
+export type AdminProjectPitchesIndexQueryParams = {
```

Comment:
I didn't expect to see this - didn't we get rid of `admin` routes? I'm also surprised to see `pageNumber` instead of `page[number]` in these. Is your current v3 branch up to date with your PR and running the latest code?